cpp/src/phonenumbers/shortnumberinfo.h

```diff
@@ -44,9 +44,6 @@ class PhoneNumberUtil;
 
 class ShortNumberInfo {
  public:
-  ShortNumberInfo();
-  ~ShortNumberInfo();
```

Comment:
By removing these public constructor & destructor, the public API is changing. I don't think we can do that since the library is widely used.